src/object.rs

```diff
@@ -566,6 +566,11 @@ pub trait ObjectExt: IsA<Object> {
     fn has_property<'a, N: Into<&'a str>>(&self, property_name: N, type_: Option<Type>) -> Result<(), BoolError>;
     fn get_property_type<'a, N: Into<&'a str>>(&self, property_name: N) -> Option<Type>;
 
+    fn block_signal(&self, handler_id: &SignalHandlerId);
+    fn unblock_signal(&self, handler_id: &SignalHandlerId);
+    fn stop_signal_emission(&self, signal_id: u32, detail: glib_ffi::GQuark);
+    fn stop_signal_emission_by_name(&self, signal_name: &str);
+
     fn connect<'a, N, F>(&self, signal_name: N, after: bool, callback: F) -> Result<SignalHandlerId, BoolError>
```

Comment:
Should probably also add a `disconnect` here for completeness

Reply:
Done.